paddle/fluid/operators/detection/distribute_fpn_proposals_op.h

```diff
@@ -28,6 +28,21 @@ namespace operators {
 
 const int kBoxDim = 4;
 
+inline std::vector<size_t> get_lod_from_rois_num(const Tensor* rois_num) {
```

Comment:
Note  code style for function-name  https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Done, thx